src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,144 @@
+package calculator;
+
+import java.util.*;
+
+/**
+ * Calculator 클래스는 계산기의 기능을 수행하는 클래스로,
+ * 입력, 데이터 검증, 사칙연산 수행, 결과값 출력을 메서드로 가진다.
+ */
+
+public class Calculator {
+    /**
+     * INPUT_MESSAGE는 상수 문자열로, 입력을 받을 때 출력되는 메세지를 저한다.
+     */
+    private final static String INPUT_MESSAGE = "계산할 수식을 입력해주세요. (숫자와 연산자는 공백으로 구분)";
+    /**
+     * ARITHMETIC_ERROR_MESSAGE는 상수 문자열로, 빈 문자열을 받을 때 출력되는 메세지를 저장한다.
+     */
+    private final static String EMPTY_ERROR_MESSAGE = "빈 문자열입니다. 다시 입력해주세요.";
+    /**
+     * ARITHMETIC_ERROR_MESSAGE 상수 문자열로, 수학 관련 예외(여기서는 0으로 나누는 경우이다) 발생 시 출력되는 메세지를 저장한다.
+     */
+    private final static String ARITHMETIC_ERROR_MESSAGE = "0으로 나눌 수 없습니다. 다시 입력해주세요.";
+    /**
+     * ILLEGAL_EQUATION_ERROR_MESSAGE는 상수 문자열로, 잘못된 수식이 연산 중 예외를 발생시킬 때에 출력되는 메세지를 저장한다.
+     */
+    private final static String ILLEGAL_EQUATION_ERROR_MESSAGE = "잘못된 수식이 입력되었습니다. 다시 입력해주세요.";
+
+    /**
+     * start 메서드는 계산기의 메인 동작을 수행한다.
+     * 올바르지 않은 입력이 들어오면 입력을 다시 요청하고,
+     * 올바른 값이 들어오면 값을 출력해준다.
+     * 이 때 while을 통한 루프 구조 대신,
+     * 자기 자신을 재귀적으로 호출하여 올바른 값이 나올 까지 실행하도록 설계하였다.
+     * <p>
+     * Exception의 경우, IllegalArgumentException(enum 접근시 잘못된 값이 들어가는 경우),
+     * NumberFormatException(잘못된 숫자가 들어가는 경우, IllegalArgumentException의 하위 클래스라 기재하지 않음),
+     * InputMismatchException(잘못된 연산자 문자열이 들어가는 경우) 을 묶어서 처리한다.
+     * 세 경우 모두 유효하지 않은 문자열이고, 별도로 나누는 것이 의미없다고 생각하였기때문이다.
+     *
+     * @throws ArithmeticException      : 만약 0으로 나누기 오류가 발생하면 예외처리한다
+     * @throws IllegalArgumentException : 만약 입력받은 숫자가 유효하지 않은 값이라면 예외처리한다
+     */
+    public void start() {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    private final static Scanner scanner = new Scanner(System.in);
```
static 변수로 바꾸는게 좋겠어요
메서드가 호출될 때마다 scanner 객체를 새로 만들거에요

Reply:
그렇네요 이렇게 짜면 수정한것이 의미가 없어지네요!! 감사합니다